### PR TITLE
tuned task #202 for IBM binding XML rules

### PR DESCRIPTION
**Pull request by mareknovotny**
result from discussion on https://tree.taiga.io/project/lincolnthree-jboss-migration/task/202

### CODE REVIEW

File: rules-reviewed/jboss-eap/websphere/websphere-xml.windup.xml [hunk 1]
```diff
@@ -75,6 +75,9 @@
                     </hint>
                 </iteration>
             </perform>
+            <where param="file_ext">
+                <matches pattern="xml|xmi"/>
```
Comment (jsight) [inline rules-reviewed/jboss-eap/websphere/websphere-xml.windup.xml]: Is there some reason to think that this filename could end in "xml"?

Comment (mareknovotny) [inline rules-reviewed/jboss-eap/websphere/websphere-xml.windup.xml]: That is mentioned at hhttp://www-01.ibm.com/support/knowledgecenter/SS7K4U_6.1.0/com.ibm.websphere.zseries.doc/info/zseries/ae/crun_app_bindings.html?cp=SS7K4U_6.1.0%2F1-7-7-2-0 and I found more hits on ibm's websphere forum

Comment (jsight) [inline rules-reviewed/jboss-eap/websphere/websphere-xml.windup.xml]: Oh ok, good find. :)

Comment (mareknovotny) [inline rules-reviewed/jboss-eap/websphere/websphere-xml.windup.xml]: btw look at taiga task #202 to read all my comments about this

**Pull request merged**